packages/app/src/specs/flaky-badge/FlakySpecSummary.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div
+    class="border-t-4px min-w-200px w-full max-w-400px grid p-4 gap-4 grid-cols-1 justify-items-center"
+    :class="accentClass"
+    data-cy="flaky-spec-summary"
+  >
+    <SpecNameDisplay
+      :spec-file-name="specName"
+      :spec-file-extension="specExtension"
+    />
+    <div class="flex flex-row w-full text-size-14px justify-center items-center">
+      <component :is="severityIcon" />
+      <span
+        class="font-medium ml-2"
+        :class="textClass"
+      >{{ severityLabel }}</span>
+      <span
+        class="ml-4"
+        data-cy="flaky-rate"
+      >{{ t('specPage.flaky.flakyRate', [flakyRate]) }}</span>
+    </div>
+
+    <div class="w-full grid text-gray-700 text-size-14px gap-2 grid-cols-2 justify-items-center">
+      <span data-cy="flaky-runs">{{ t('specPage.flaky.flakyRuns', [totalFlakyRuns, totalRuns]) }}</span>
+      <span data-cy="last-flaky">{{ t('specPage.flaky.lastFlaky', [lastFlaky]) }}</span>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+
+import { computed, FunctionalComponent, ref, shallowRef, watch } from 'vue'
+import { useI18n } from '@cy/i18n'
+import SpecNameDisplay from '../SpecNameDisplay.vue'
+import LowRateIcon from '~icons/cy/rate-low_x16'
+import MediumRateIcon from '~icons/cy/rate-medium_x16'
+import HighRateIcon from '~icons/cy/rate-high_x16'
+import ceil from 'lodash/ceil'
+
+const { t } = useI18n()
+
+const props = defineProps<{
+  specName: string
+  specExtension: string
+  severity: 'low' | 'medium' | 'high'
+  totalFlakyRuns: number
+  totalRuns: number
+  lastFlaky: number
+  dashboardUrl: string
+}>()
+
+const accentClass = ref<string | null>(null)
+const textClass = ref<string | null>(null)
+const severityLabel = ref<string | null>(null)
+const severityIcon = shallowRef<FunctionalComponent | null>(null)
+const flakyRate = computed(() => {
+  if (props.totalFlakyRuns <= 0 || props.totalRuns <= 0) {
+    return 0
+  }
+
+  const rawRate = props.totalFlakyRuns / props.totalRuns * 100
+
+  if (rawRate > 99 && rawRate < 100) {
+    return 99
```

Comment:
Wouldn't a value of 99.5 round up to 100 anyway? Why do we need this check?

Reply:
Normally we do a round up, _unless_ the value is between 99 and 100 in which case we round down here

Reply:
Maybe it would be helpful to add a comment like
`Only show 100 if rawRate is actually 100`

Reply:
I don't understand why we have specific logic for > 99 and < 100 :thinking: 

why?

Reply:
It was a request from Product in order to match behavior on the dashboard ([documented in the PRD](https://docs.google.com/document/d/1u5cyqB6uDcGnhDnex49XKODJzZKdBpYr1xbgovS-bC8/edit#heading=h.shvspemfqn1w)). Not sure why it was done that way there 🤷

Reply:
Okay, let's just roll with it :zap: